lib/posix-process/exportsyms.uk

```diff
@@ -81,3 +81,5 @@ uk_syscall_r_clone
 uk_syscall_e_clone
 uk_syscall_r_clone3
 uk_syscall_e_clone3
+uk_syscall_r_fork
```

Comment:
I wouldn't export these. Placing them here is misleading since someone might think is a real fork. Removing them shouldn't affect the compilation

Reply:
It does actually affect the compilation process. When adding `fork` to the provided syscalls in [`Makefile.uk`](https://github.com/unikraft/unikraft/pull/642/files#diff-bef8ca6eca25e3c6b4fd6c3c03a0a968d7cb46d2e57f572fc576b91bdf75656cR27), it also ends up in `build/libsyscall_shim/include/uk/bits/provided_syscalls.h`, where `HAVE_uk_syscall_fork` is defined.

When `uk_syscall_e_fork` and `uk_syscall_r_fork` are not exported, then linking fails. One such error consists of the fact that these 2 cannot be found in [`uk_syscall6`](https://github.com/unikraft/unikraft/blob/staging/lib/syscall_shim/gen_uk_syscall6.awk)  although `HAVE_uk_syscall_fork` is defined.

I also feel that they should not be exported, but right now I cannot think of a better way of implementing this test. Do you have any ideas?

Reply:
You're right the compilation fails if we don't export them. In that case we can add a comment like "PLEASE NOTE THIS IS NOT A REAL FORK". You can add them with `#` in exportsyms.uk. Can you please check if this compiles for you as well?